src/httpserver.cpp

```diff
@@ -467,7 +467,7 @@ void StopHTTPServer()
     boundSockets.clear();
     if (eventBase) {
         LogPrint(BCLog::HTTP, "Waiting for HTTP event thread to exit\n");
-        threadHTTP.join();
+        if (g_thread_http.joinable()) g_thread_http.join();
```

Comment:
In commit "http: Avoid crash when g_thread_http was never started" (faf838bf1cae719c46211ea603848f367ee427cf)

I found Waldo!